syz-ci/syz-ci.go

```diff
@@ -95,6 +96,9 @@ type Config struct {
 	BisectBinDir    string           `json:"bisect_bin_dir"`
 	Ccache          string           `json:"ccache"`
 	Managers        []*ManagerConfig `json:"managers"`
+
+	// Not changeable via config file.
+	Debug bool
```

Comment:
It is changeable via config. The `json` field tag does not enable json, it only changes name how it is represented in json.
It may be cleaner to just pass as a flag to createManager.

Reply:
fixed, thanks